To fix issue found in #4

Run pip install -r requirements.txt
Collecting selenium==4.5.0
  Downloading selenium-4.5.0-py3-none-any.whl (995 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 995.2/995.2 kB 63.1 MB/s eta 0:00:00
Collecting requests==2.2[8](https://github.com/Zenious/BadmintonCourtAvailChecker/actions/runs/3221407477/jobs/5269301197#step:5:9).1
  Downloading requests-2.28.1-py3-none-any.whl (62 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 62.8/62.8 kB 18.5 MB/s eta 0:00:00
Collecting beautifulsoup4==4.11.1
  Downloading beautifulsoup4-4.11.1-py3-none-any.whl (128 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 128.2/128.2 kB 35.3 MB/s eta 0:00:00
Collecting python-dotenv==0.21.0
  Downloading python_dotenv-0.21.0-py3-none-any.whl (18 kB)
Collecting urllib3[socks]~=1.26
  Downloading urllib3-1.26.12-py2.py3-none-any.whl (140 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 140.4/140.4 kB 34.4 MB/s eta 0:00:00
Collecting certifi>=2021.10.8
  Downloading certifi-2022.[9](https://github.com/Zenious/BadmintonCourtAvailChecker/actions/runs/3221407477/jobs/5269301197#step:5:10).24-py3-none-any.whl (161 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 161.1/161.1 kB 42.0 MB/s eta 0:00:00
Collecting trio~=0.17
  Downloading trio-0.22.0-py3-none-any.whl (384 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 384.9/384.9 kB 58.1 MB/s eta 0:00:00
Collecting trio-websocket~=0.9
  Downloading trio_websocket-0.9.2-py3-none-any.whl (16 kB)
Collecting idna<4,>=2.5
  Downloading idna-3.4-py3-none-any.whl (61 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.5/61.5 kB 20.0 MB/s eta 0:00:00
Collecting charset-normalizer<3,>=2
  Downloading charset_normalizer-2.1.1-py3-none-any.whl (39 kB)
Collecting soupsieve>1.2
  Downloading soupsieve-2.3.2.post1-py3-none-any.whl (37 kB)
Collecting attrs>=19.2.0
  Downloading attrs-22.1.0-py2.py3-none-any.whl (58 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 58.8/58.8 kB 19.3 MB/s eta 0:00:00
Collecting outcome
  Downloading outcome-1.2.0-py2.py3-none-any.whl (9.7 kB)
Collecting sniffio
  Downloading sniffio-1.3.0-py3-none-any.whl ([10](https://github.com/Zenious/BadmintonCourtAvailChecker/actions/runs/3221407477/jobs/5269301197#step:5:11) kB)
Collecting async-generator>=1.9
  Downloading async_generator-1.10-py3-none-any.whl (18 kB)
Collecting sortedcontainers
  Downloading sortedcontainers-2.4.0-py2.py3-none-any.whl (29 kB)
Collecting exceptiongroup>=1.0.0rc9
  Downloading exceptiongroup-1.0.0rc9-py3-none-any.whl (12 kB)
Collecting wsproto>=0.14
  Downloading wsproto-1.2.0-py3-none-any.whl (24 kB)
Collecting PySocks!=1.5.7,<2.0,>=1.5.6
  Downloading PySocks-1.7.1-py3-none-any.whl (16 kB)
Collecting h[11](https://github.com/Zenious/BadmintonCourtAvailChecker/actions/runs/3221407477/jobs/5269301197#step:5:12)<1,>=0.9.0
  Downloading h11-0.14.0-py3-none-any.whl (58 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 58.3/58.3 kB 16.8 MB/s eta 0:00:00
Installing collected packages: sortedcontainers, urllib3, soupsieve, sniffio, python-dotenv, PySocks, idna, h11, exceptiongroup, charset-normalizer, certifi, attrs, async-generator, wsproto, requests, outcome, beautifulsoup4, trio, trio-websocket, selenium
Successfully installed PySocks-1.7.1 async-generator-1.10 attrs-22.1.0 beautifulsoup4-4.11.1 certifi-2022.9.24 charset-normalizer-2.1.1 exceptiongroup-1.0.0rc9 h11-0.14.0 idna-3.4 outcome-1.2.0 python-dotenv-0.21.0 requests-2.28.1 selenium-4.5.0 sniffio-1.3.0 sortedcontainers-2.4.0 soupsieve-2.3.2.post1 trio-0.22.0 trio-websocket-0.9.2 urllib3-1.26.[12](https://github.com/Zenious/BadmintonCourtAvailChecker/actions/runs/3221407477/jobs/5269301197#step:5:13) wsproto-1.2.0
************* Module index
index.py:21:0: E1101: Instance of 'WebDriver' has no 'find_element_by_id' member (no-member)
index.py:22:0: E1101: Instance of 'WebDriver' has no 'find_element_by_id' member (no-member)
index.py:24:0: E1101: Instance of 'WebDriver' has no 'find_element_by_id' member (no-member)
index.py:29:[16](https://github.com/Zenious/BadmintonCourtAvailChecker/actions/runs/3221407477/jobs/5269301197#step:5:17): E1101: Instance of 'WebDriver' has no 'find_element_by_id' member (no-member)

-----------------------------------
Your code has been rated at 6.00/10

Error: Process completed with exit code 2.

--- a/index.py
+++ b/index.py
@@ -6,6 +6,7 @@ from bs4 import BeautifulSoup
 from dotenv import load_dotenv
 from selenium import webdriver
 from selenium.webdriver.support.ui import Select
+from selenium.webdriver.common.by import By
 
 load_dotenv()
 
@@ -18,15 +19,15 @@ EPOCH_TIME = 1633622400
 driver = webdriver.Chrome('./chromedriver')
 driver.get('https://members.myactivesg.com/auth')
 
-driver.find_element_by_id("email").send_keys(USERNAME)
-driver.find_element_by_id("password").send_keys(PASSWORD)
+driver.find_element(By.ID, "email").send_keys(USERNAME)
+driver.find_element(By.ID, "password").send_keys(PASSWORD)
 time.sleep(2)
-driver.find_element_by_id("btn-submit-login").click()
+driver.find_element(By.ID, "btn-submit-login").click()
 
 time.sleep(1)
 driver.get('https://members.myactivesg.com/facilities/view/activity/18/venue/292')
 time.sleep(1)
-select = Select(driver.find_element_by_id('facVenueSelection'))
+select = Select(driver.find_element(By.ID, 'facVenueSelection'))
 for option in (select.options):
     if not option.get_attribute("value"):
         continue

